specification/edgeorder/resource-manager/Microsoft.EdgeOrder/preview/2020-12-01-preview/edgeorder.json

```diff
@@ -376,17 +376,24 @@
             "required": false,
             "type": "string"
           },
+          {
+            "name": "$expand",
+            "in": "query",
+            "description": "$expand is supported on device details parameter for order item, which provides details on the devices of the product.",
```

Comment:
We support expand on forward shipping details and reverse shipping details parameters as well. Can you please check